MoviesforSale/app/src/main/java/com/ezequielc/moviesforsale/MovieAdapter.java

```diff
@@ -0,0 +1,51 @@
+package com.ezequielc.moviesforsale;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+/**
+ * Created by student on 11/2/16.
+ */
+
+public class MovieAdapter extends RecyclerView.Adapter<MovieViewHolder> {
+    private List<Movies> mMoviesList;
+    private MovieListFragment.OnMovieSelectedListener mListener;
+
+    public MovieAdapter(List<Movies> moviesList, MovieListFragment.OnMovieSelectedListener listener) {
+        mMoviesList = moviesList;
+        mListener = listener;
+    }
+
+    @Override
+    public MovieViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        return new MovieViewHolder(LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.movie_list_layout, parent, false));
```

Comment:
I recommend naming your XML file `movie_list_item_layout` rather than `movie_list_layout` so it's clear it describes only one item in a list